src/Logger/Adapter/LogOwl.php

```diff
@@ -24,6 +24,22 @@ class LogOwl extends Adapter
      */
     protected string $logOwlHost;
 
+    /**
+     * LogOwl constructor.
+     *
+     * @param  string  $ticket
+     * @param  string  $host
+     */
+    public function __construct(string $ticket, string $host)
```

Comment:
Host should be optional, `string $host = 'https://api.logowl.io/logging/'`

Reply:
done 👍